dfc/REST.go

```diff
@@ -72,13 +72,14 @@ const (
 // TODO: sort and some props are TBD
 // GetMsg represents properties and options for get requests
 type GetMsg struct {
-	GetWhat       string `json:"what"`        // "config" | "stats" ...
-	GetSort       string `json:"sort"`        // "ascending, atime" | "descending, name"
-	GetProps      string `json:"props"`       // e.g. "checksum, size" | "atime, size" | "ctime, iscached" | "bucket, size"
-	GetTimeFormat string `json:"time_format"` // "RFC822" default - see the enum below
-	GetPrefix     string `json:"prefix"`      // object name filter: return only objects which name starts with prefix
-	GetPageMarker string `json:"pagemarker"`  // AWS/GCP: marker
-	GetPageSize   int    `json:"pagesize"`    // maximum number of entries returned by list bucket call
+	GetWhat       string `json:"what"`           // "config" | "stats" ...
+	GetSort       string `json:"sort"`           // "ascending, atime" | "descending, name"
+	GetProps      string `json:"props"`          // e.g. "checksum, size" | "atime, size" | "ctime, iscached" | "bucket, size"
+	GetTimeFormat string `json:"time_format"`    // "RFC822" default - see the enum below
+	GetPrefix     string `json:"prefix"`         // object name filter: return only objects which name starts with prefix
+	GetPageMarker string `json:"pagemarker"`     // AWS/GCP: marker
+	GetPageSize   int    `json:"pagesize"`       // maximum number of entries returned by list bucket call
+	GetTargetURL  bool   `json:"get_target_url"` // if should return the target URL for each object on list bucket
```

Comment:
nitpick: I think 'get' should be removed from json:get_target_url. All others do not start from get

Reply:
how's `return_target_url`?

Reply:
I want to be explicit here that the property **isn't** a string representing the actual target's URL, rather, a boolean representing whether a target URL should be returned.

Reply:
By the way, why did you add a new property to request instead of adding a new property to return? I expected that you add GetPropsTargetURL and send it as a member of GetProps field.
TargetURL is a new field of object property list returned by listbucket. It looks strange to have all other fields in GetProps but extract TaregtURL prop to a separate bool in request body

Reply:
Slacked ur bro